Backend/app/rider-platform/rider-app/Main/src/API/UI/Search.hs

```diff
@@ -177,7 +250,7 @@ oneWaySearch personId bundleVersion clientVersion device req = do
     merchantConfigs <- CMC.findAllByMerchantId person.merchantId
     SMC.updateSearchFraudCounters personId merchantConfigs
     mFraudDetected <- SMC.anyFraudDetected personId person.merchantId merchantConfigs
-    whenJust mFraudDetected $ \mc -> SMC.blockCustomer personId (Just mc.id)
+    whenJust mFraudDetected $ \mc -> SMC.takeAction personId (Just mc.id) mc.shouldSimulate
```

Comment:
should we use keywords like "fraud"? as the code is now opensource @shanm-16  @piyushKumar-1